### ISSUE TITLE
Regression in 6.6.1

### ISSUE THREAD

**Issue by sigio** [ansible/ansible-lint#2475]
##### Summary

Ansible-lint version 6.6.1 fails on line 1 of a file which validated fine with 6.6.0. We run ansible-lint as part of out github CI workflows, and runs started failing today, on files that haven't had any changes.

Run ansible-lint -p ansible-etc/site.yaml
  
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: ansible-etc/site.yaml
WARNING  Listing 1 violation(s) that are fatal
ansible-etc/site.yaml:1: syntax-check: Ansible syntax check failed.
Error: syntax-check Ansible syntax check failed.

##### Issue Type

- Regression

##### Ansible and Ansible Lint details
 
```
ansible [core 2.13.4]
  config file = None
  configured module search path = ['/home/runner/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/runner/.local/lib/python3.8/site-packages/ansible
  ansible collection location = /home/runner/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/runner/.local/bin/ansible
  python version = 3.8.10 (default, Jun 22 2022, 20:18:18) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True
ansible-lint 6.6.1 using ansible 2.13.4
 ```
##### OS / ENVIRONMENT

Github workflow running on ubuntu 20.04 image with ansible-lint from pip

**Comment by sigio** [comment]
The first lines of the playbook ansible-lint is complaining about are:

# SITE Playbook

---

- name: Site Global Playbook
  hosts: all
  order: rtl_sorted
  gather_facts: true
  become: true
  become_user: root
  serial: "{{ serial_number | default( ansible_check_mode|ternary(100, 20)) }}"
  pre_tasks:
.....

**Comment by sigio** [comment]
Ignore
